### DISCUSSION TITLE
打不开资源

### DISCUSSION THREAD

**Discussion by zxkke** [Hiram-Wong/ZyPlayer#327] [General]
![屏幕截图 2024-09-18 131417](https://github.com/user-attachments/assets/3ba47a23-76d5-47b5-9eb9-678cef65b4b7)
![屏幕截图 2024-09-18 131305](https://github.com/user-attachments/assets/cd040b89-b3ca-464d-a56f-9ea0abd46c8e)

**Comment by Hiram-Wong** [comment]
请须知 官源需要解析
